tendrl/ceph_integration/ceph.py

```diff
@@ -392,45 +396,97 @@ def ceph_command(cluster_name, command_args):
     }
 
 
-def rbd_command(cluster_name, command_args, pool_name=None):
-    """Run a rbd CLI operation directly.  This is a fallback to allow
-
-    manual execution of arbitrary commands in case the user wants to
-
-    do something that is absent or broken in Calamari proper.
-
-    :param pool_name: Ceph pool name, or None to run without --pool argument
-
-    :param command_args: Command line, excluding the leading 'rbd' part.
-
-    """
-
-    if pool_name:
-        args = ["rbd", "--pool", pool_name, "--cluster", cluster_name] + \
-            command_args
+def rbd_operation(cluster_name, operation, attr, pool_name, result, queue):
+    import rados
+    cluster = rados.Rados(name=RADOS_NAME,
+                          clustername=cluster_name,
+                          conffile='')
+    cluster.connect()
+    
+    try:
+        ioctx = cluster.open_ioctx(pool_name)
+        try:
+            if operation == "create":
+                rbd_inst = rbd.RBD()
+                rbd_inst.create(ioctx, attr["name"], int(attr["size"]))
```

Comment:
@GowthamShanmugam you should mention the UI folks if there's some assumption like above 

cc: @Tendrl/tendrl_frontend